pkg/cmd/allocsim/main.go

```diff
@@ -496,7 +499,8 @@ func main() {
 		os.Exit(exitStatus)
 	}()
 
-	c.Start("allocsim", *workers, os.Args[0], nil, flag.Args(), perNodeArgs)
+	allNodeArgs := append(flag.Args(), "--vmodule=allocator=1")
+	c.Start("allocsim", *workers, os.Args[0], nil, allNodeArgs, perNodeArgs)
```

Comment:
Not necessary, but it seems like a good default for a tool whose purpose is to test out the allocator.